rpcs3/rpcs3qt/welcome_dialog.cpp

```diff
@@ -51,33 +46,44 @@ welcome_dialog::welcome_dialog(std::shared_ptr<gui_settings> gui_settings, bool
 		)"
 	).arg(gui::utils::get_link_style()));
 
+#ifdef __APPLE__
+	ui->create_applications_menu_shortcut->setText(tr("&Create Launchpad shortcut"));
+	ui->use_dark_theme->setVisible(false);
+	ui->use_dark_theme->setEnabled(false);
+#else
+	#ifndef _WIN32
```

Comment:
Wrong indentation

Reply:
Ok, changing. That is usually my codestyle